underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -365,6 +380,21 @@ private ListObjectsV2Result getObjectListingChunk(ListObjectsV2Request request)
     return result;
   }
 
+  // Get next chunk of listing result
+  private ObjectListing getObjectListingChunkV1(ListObjectsRequest request) {
+    ObjectListing result;
+    try {
+      // Query S3 for the next batch of objects
+      result = mClient.listObjects(request);
+      // Advance the request continuation token to the next set of objects
+      request.setMarker(result.getNextMarker());
+    } catch (AmazonClientException e) {
+      LOG.error("Failed to list path {}", request.getPrefix(), e);
+      result = null;
```

Comment:
The method javadoc says ` @return chunked object listing, or null if key is not found`, so it seems strange to return `null` whenever any sort of exception happens. Is there a way to tell the difference between the key not existing (return null) and an unexpected error happening (propagate exception)?